src/ressa/context.rs

```diff
@@ -10,31 +10,34 @@ const RESOLVES_TO: &'static str = "???";
 const TRANSIENT: &'static str = "";
 
 /// Context used by the Parser, storing local variables (#{varname}) and objects/tags
-#[derive(Default, Debug, Clone, Any, PartialEq)]
+#[derive(Default, Debug, Clone, Any)]
 pub struct ParserContext {
-    variables: ContextData,
-    local_variables: HashMap<String, String>,
+    objectlike_data: HashMap<String, Object>,
+    variables: HashMap<String, String>,
     pub frame_number: i32,
 }
 
-impl Into<ContextData> for ParserContext {
-    fn into(self) -> ContextData {
-        self.variables
+impl Into<RessaResult> for ParserContext {
+    fn into(self) -> RessaResult {
+        self.objectlike_data
             .into_iter()
             .filter(|(_, val)| !val.contains_key(RESOLVES_TO) && !val.contains_key(TRANSIENT))
+            .map(|obj| (obj.0, obj.1.into_inner()))
```

Comment:
Sort of nitpicking here (so not really necessary to change it), but I like to avoid using the dot notation on tuples and destructure them instead for a bit more readability
```rust
.map(|(name, value)| (name, value.into_inner()))
```